test/runtests.jl

```diff
@@ -29,7 +29,7 @@ pms_path = joinpath(dirname(pathof(PowerModels)), "..")
 
 ipopt_solver = JuMP.with_optimizer(Ipopt.Optimizer, tol=1e-6, print_level=0)
 cbc_solver = JuMP.with_optimizer(Cbc.Optimizer, logLevel=0)
-scs_solver = JuMP.with_optimizer(SCS.Optimizer, max_iters=10000, verbose=0)
+scs_solver = JuMP.with_optimizer(SCS.Optimizer, max_iters=10000, eps=1e-6, verbose=0)
```

Comment:
For my edification, what is the need for `eps`?

Reply:
I added this at some point to get a closer match for some test case, but it does not seem necessary for the current unit tests. I will remove it.